backend/app/config.py

```
@@ -0,0 +1,42 @@
+from typing import Optional
+
+
+class Settings(BaseSettings):
+    """Application settings."""
+    
+    # App
+    APP_NAME: str = "SolFoundry"
+    DEBUG: bool = True
+    
+    # JWT
+    SECRET_KEY: str = "your-secret-key-change-in-production"
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)